doc/no-runtime.md

```diff
@@ -14,10 +14,7 @@ If for some reason you want to use your own network layer and don't want to use
 ```java
     okhttp3.Response httpResponse = ...;
 
-    // if you have custom scalar types, provide proper instance of ScalarTypeAdapters with your own custom adapters
-    ScalarTypeAdapters scalarTypeAdapters = new ScalarTypeAdapters(Collections.<ScalarType, CustomTypeAdapter>emptyMap());
```

Comment:
I would keep this note. I think 90% of our users have at least one CustomTypeAdapter for Dates.